README.md

```diff
@@ -1,101 +1,247 @@
+  and its tests.  You will need to set `ENABLE_FleCSI=True` and add
+  the FleCSI and FleCSI-sp install paths to the `CMAKE_PREFIX_PATH`;
+  see examples below.  Both FleCSI packages are under constant
+  development.  This version of portage is known to work with hash
+  `374b56b` of the FleCSI _stable_ branch, and hash `e78c594` of the
+  FleCSI-SP _stable_ branch.
+
+The [documentation](http://portage.lanl.gov) is built using doxygen (1.8+).
+
+For more details regarding CMake settings, see
+the [documentation](http://portage.lanl.gov) page.
+
+### Installing
+
+In the simplest case where you have the appropriate versions mentioned
+above and Boost and LAPACKE are in the usual locations that CMake
+searches, then the build step is:
+
+```sh
+portage $ mkdir build
+portage $ cd build
+portage/build $ cmake -DENABLE_APP_TESTS=True ..
+portage/build $ make
+```
+
+This compiles the serial code and about a dozen application tests.  To
+run the tests, simply execute
+
+```sh
+portage/build $ make tests
+```
+
+If you wish to install the code into the `CMAKE_INSTALL_PREFIX` then
+simply execute
+```sh
+portage/build $ make install
+```
+
+To build the documentation, one would configure with the
+`-DENABLE_DOXYGEN=True` flag, and then `make doxygen`.
+
+See the examples below, or the
+[documentation](http://portage.lanl.gov) for more build instructions.
+
+# License
+
+This project is licensed under a modified 3-clause BSD license - see
+the [LICENSE](https://github.com/laristra/portage/blob/master/LICENSE)
+file for details.
+
+# Release
+
+This software has been approved for open source release and has been
+assigned **LA-CC-16-084**.
+
+----
+----
+
+# Example builds
+
+Below we list copy & paste instructions for several local machines; we
+have a script that parses this README file to execute the examples
+below to ensure they build.
+
+## Darwin
+
+Execute the following from the portage root directory:
+
+```c++
+# machine=darwin-fe
+module load openmpi/2.0.1-intel_17.0.0 cmake/3.7.1
+JALI_INSTALL_PREFIX=/projects/ngc/private/jali/0.9.8-intel-17.0.0-openmpi-2.0.1
+TPL_INSTALL_PREFIX=/projects/ngc/private/jali-tpl/1.0.9-intel-17.0.0-openmpi-2.0.1
+mkdir build
+cd build
+cmake \
+    -D CMAKE_C_COMPILER=`which mpicc` \
+    -D CMAKE_CXX_COMPILER=`which mpiCC` \
+    -D CMAKE_BUILD_TYPE=Debug \
+    -D ENABLE_UNIT_TESTS=True \
+    -D ENABLE_APP_TESTS=True \
+    -D ENABLE_MPI=True \
+    -D Jali_DIR:FILEPATH=$JALI_INSTALL_PREFIX/lib \
+    -D Boost_INCLUDE_DIR:PATH=$TPL_INSTALL_PREFIX/include \
+    ..
+make -j16
+ctest -j16 --output-on-failure
+```
+
+## Moonlight
+
+Execute the following from the portage root directory:
+
+```c++
+# machine=ml-fey
+module load intel/17.0.1 openmpi/1.10.5 cmake
+JALI_INSTALL_PREFIX=/usr/projects/ngc/private/jali/0.9.8-intel-17.0.1-openmpi-1.10.5
+mkdir build
+cd build
+cmake \
+    -D CMAKE_C_COMPILER=`which mpicc` \
+    -D CMAKE_CXX_COMPILER=`which mpiCC` \
```

Comment:
The `which mpicc` isn't needed, see 95327cb9e559ffeb1bd55c1d5629bbb5a19c1d66.

Reply:
Ah, good to know.